### PR TITLE
code: Generate download cards dynamically

### PR DESCRIPTION
**Pull request by aperezdc**
Replace the static cards with source package downloads with dynamically generated ones which always link to the most recent stable and unstable downloads and their release notes.

----

Site preview: https://igalia.github.io/wpewebkit.org/aperezdc/touchup-dl-cards/

### CODE REVIEW

**Comment by aperezdc** [comment]
Before, without this PR:

before

After, with this PR applied:

after

The version badges link to the tarballs, and the secondary badge with the :information_source: icons link to the release notes for that version.

**Comment by bkardell** [comment]
Can we use the sr-only spans inside the anchors to give them names that will be discoverable/sensible? Download v1.2.3 or release notes for v1.2.3?

**Comment by aperezdc** [comment]
> Can we use the sr-only spans inside the anchors to give them names that will be discoverable/sensible? Download v1.2.3 or release notes for v1.2.3?

@bkardell: Of course. I am far from knowing much about a11y, so it's not surprising this bit could be better :innocent: 

I'll update the MR along the day with your suggestion, and also add a `title=` attribute to the `<a>` elements, so the browser can show a tooltip with a text like `Download Cog version 1.2.3` while hovering the badges with the mouse.

Is there any quick way to check things that would be better to load the page in [Lynx](https://invisible-island.net/lynx/)?

**Comment by aperezdc** [comment]
Yes, I think that the `title=` attributes are a great idea, so we get this for free on browsers that use them as tooltips (which is most desktop browsers):

Screen capture of a the tooltip shown when hovering a download badge

**Comment by aperezdc** [comment]
Changes pushed. There is a `<h4 class="sr-only">` for titling the package downloads section, and in the end I have also changed the card header elements from `<div>` to `<h5>` which I think better reflects the semantics of the content.

This is how the source downloads look like in Lynx, which I understand gives a reasonably decent idea of how a screen reader would “see” the content:

Screenshot of the Lynx browser showing the source package downloads

**Comment by bkardell** [comment]
> Is there any quick way to check things that would be better to load the page in Lynx?

Short of loading it, you mean?

**Comment by aperezdc** [comment]
> > Is there any quick way to check things that would be better to load the page in Lynx?
> 
> Short of loading it, you mean?

Checking how things look like in Lynx is not a problem at all. My doubt was about alternatives, let me rephrase: is there some other tool other than Lynx to check how a11y technologies will “see” the content which would be closer to what e.g. an actual screen reader will do? (Without going through the effort of configuring and learning to use an screen reader).
